Modules/Microsoft365DSC/DSCResources/MSFT_SPOAccessControlSettings/MSFT_SPOAccessControlSettings.schema.mof

```diff
@@ -22,4 +22,5 @@ class MSFT_SPOAccessControlSettings : OMI_BaseResource
     [Write, Description("Path to certificate used in service principal usually a PFX file.")] String CertificatePath;
     [Write, Description("Thumbprint of the Azure Active Directory application's authentication certificate to use for authentication.")] String CertificateThumbprint;
     [Write, Description("Managed ID being used for authentication.")] Boolean ManagedIdentity;
+    [Write, Description("Blocks or limits access to SharePoint and OneDrive content from un-managed devices.")] Boolean ConditionalAccessPolicy;
```

Comment:
The methods signature define it as String with a list of accepted values, whereas the schema defines it as Boolean. Can we please make sure the schema is correct and that it lists the  proper ValueMap and Values list. Thanks

Reply:
thank you for pointing that out. I did miss that part to correctly include. I hope my fix does correct his addressed issue?